packages/create-puck-app/index.js

```diff
@@ -77,10 +77,17 @@ program
         required: true,
         default: "next",
       },
+      {
+        type: "confirm",
+        name: "initializeGit",
+        message: "Do you want to initialize and commit using Git?",
+        default: true,
+      },
```

Comment:
I'd still prefer to auto initialize, as I believe this is pretty standard in most create-* projects (although happy to be proven wrong).